library/spdm_requester_lib/libspdm_req_negotiate_algorithms.c

```diff
@@ -106,6 +106,7 @@ static libspdm_return_t libspdm_try_negotiate_algorithms(libspdm_context_t *spdm
     spdm_request_size = message_size - transport_header_size -
                         spdm_context->local_context.capability.transport_tail_size;
 
+    LIBSPDM_ASSERT(spdm_request_size >= sizeof(libspdm_negotiate_algorithms_request_mine_t));
```

Comment:
It could be smaller.

Reply:
But the `libspdm_negotiate_algorithms_request_mine_t` size buffer will be zeroed.

https://github.com/DMTF/libspdm/blob/16e56a6f177925083aed6db6149ecfa9b43ed2fe/library/spdm_requester_lib/libspdm_req_negotiate_algorithms.c#L109-L110

Reply:
I have changed the code too.